src/quo2/components/filter_tag.cljs

```diff
@@ -3,8 +3,8 @@
             [quo.theme :as theme]
             [quo.react-native :as rn]
             [quo2.components.icon :as icons]
-            [quo2.components.text :as text]
-            [quo2.components.base-tag :as base-tag]))
+            [quo2.components.markdown.text :as text]
+            [quo2.components.tags.base-tag :as base-tag]))
 
 (def themes {:light {:default  {:border-color             colors/neutral-20
```

Comment:
is there a reason why you didnt place this file in the tags directory?

Reply:
you could also confirm that you didnt leave it out under the screens directory. It seems filter-tags are missing there

Reply:
yeah I wanted to check about this component. I could not find it in the figma file. 
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=180%3A9668